src/ResponseTagger.php

```diff
@@ -89,4 +90,22 @@ public function addTags(array $tags)
 
         return $this;
     }
+
+    /**
+     * Set tags on a response
+     *
+     * @param ResponseInterface $response Original response
+     * @param bool              $replace  Whether to replace the current tags
+     *                                    on the response
+     *
+     * @return ResponseInterface          Tagged response
+     */
+    public function tagResponse(ResponseInterface $response, $replace = false)
```

Comment:
Remove this typehint (see #230) or add another method `tagSymfonyResponse` to the bundle’s ResponseTagger (previously TagHandler)?